.github/workflows/build_main_documentation.yml

```diff
@@ -9,7 +9,7 @@ on:
       package:
         required: true
         type: string
-      path_to_doc:
+      path_to_docs:
```

Comment:
I think there was a typo in the original definition, so I've updated this to reflect the variable name in the rest of the YAML